priv/repo/add_venue_slug.exs

```diff
@@ -2,6 +2,8 @@ alias CsGuide.Resources.Venue
 
 Venue.all()
 |> Enum.map(fn v ->
-  slug = Venue.create_slug(v.venue_name, v.postcode)
-  Venue.update(v, %{slug: slug})
+  if v.slug == nil do
+    slug = Venue.create_slug(v.venue_name, v.postcode)
+    Venue.update(v, %{slug: slug})
```

Comment:
This is a good fix for now. 👍 
Ideal would be to have a test that references the issue/feature. 💭